tailwind.config.js

```diff
@@ -0,0 +1,20 @@
+module.exports = {
+  purge: {
+    content: [
+      './**/*.php',
```

Comment:
Is this globbing everything including `vendor`? Might be faster to scope this on `app` and `resources` but I haven't done a strict comparison.

Reply:
hadn't thought of it. not 100% sure – but you're probably right.